### DISCUSSION TITLE
Considering the Octahedral shear strain in the multiple-surfaces plastic Iwan model

### DISCUSSION THREAD

**Discussion by tengfeideng** [thelfer/tfel#449] [General]
Dear Thomas,

I'm recently using the [Iwan](https://thelfer.github.io/tfel/web/iwan.html) Mfront model to simulate the soil behavior under dynamic loading. I read the tutorial and found the $\gamma= 2 \epsilon_{ij}, i \ne j$. I think it means the shear modulus degradation relates to the $\epsilon_{ij}$.  Now I want to consider the generalized shear strain, like Octahedral shear strain, 

 $\gamma= \frac{2}{3} [(\epsilon_{xx} - \epsilon_{yy})^2 + (\epsilon_{yy} - \epsilon_{zz})^2 + \epsilon_{xx} - \epsilon_{zz})^2+ 6\epsilon_{xy}^2 + 6\epsilon_{yz}^2 + 6\epsilon_{xz}^2] ^{1/2}$

Do you have some advice about it?  or maybe I have a misunderstanding and the shear modulus degradation relating to generalized shear strength has been considered in the MFront Iwan model. 

best

tengfei

**Comment by thelfer** [comment]
@tengfeideng The Iwan beahviour has been implementd as a set of plastic surfaces that each evolve thanks to a Prager kinematic hardening. The yield surface is defined using on the von Mises stress.

The relationship with the  shear modulus degradation is identified using shear tests, but the behaviour is fully 3D.

**Reply by tengfeideng** [reply]
Thanks a lot!
